scripts/sign.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+PROGRAM_NAME="${0##*/}"
```

Comment:
Are we missing a set -eux or similar?

Reply:
obligitory "gah, `bash`" comment.

Reply:
(@karianna's correct here @jmjaffe37)

Reply:
Are we sure that we want to include the 'x' flag? It seems to make printed output weird, Especially when we get output from `--help`

Reply:
Ah yes, `-x` is `xtrace` and I believe it's intended use is for _debugging_ so you might want to leave that one off. (You can turn it on temporarily from the command line by calling your script with `bash -x sign.sh`.)

But I'll defer to the `bash` experts in the crowd.

Reply:
@karianna @gdams

Reply:
set -eu is fine